packages/hir/package.json

```diff
@@ -11,18 +11,15 @@
   "module": "dist/modules/index.js",
   "types": "dist/commonjs/index.d.ts",
   "scripts": {
-    "prepublish": "npm run build",
-    "build": "rm -rf dist; tsc -p . & tsc -p . --module ESNext --outDir dist/modules/ --target ES2017",
+    "build": "rm -rf dist; tsc -p . && tsc -p . --module ESNext --outDir dist/modules/ --target ES2017",
+    "lint": "../../node_modules/.bin/tslint -c ../../tslint.json 'src/**/*.ts'",
```

Comment:
with binaries, can we make it so that paths are never relative to the lerna host container? Ideally a package will run on its own, and hoisting will mean that all of those tools are available to each sub-package, right?